certbot/bin/entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+until curl --output /dev/null --silent --head --fail http://"${CERT_DOMAIN}"; do
+    echo "Cannot resolve ${CERT_DOMAIN}..."
+    sleep 5
+done
+
+echo "Response from http://${CERT_DOMAIN}, attempting to obtain certificate..."
+
+run_certbot.sh -e "${ADMIN_EMAIL}" -d "${CERT_DOMAIN}" -r /challenges
```

Comment:
This only runs once right?

Reply:
You can call it multiple times to renew the cert if it already exists, but the behaviour right now is to run once to get the certificate then exit.

Reply:
K